src_base/xeve_def.h

```diff
@@ -822,7 +822,8 @@ typedef struct _XEVE_SBAC_CTX
 /* rpl structure */
 #define XEVE_MAX_NUM_REF_PICS                   21
 #define XEVE_MAX_NUM_ACTIVE_REF_FRAME           5
-#define XEVE_MAX_NUM_RPLS                       32
+
+#define XEVE_MAX_NUM_RPLS                       64
```

Comment:
Duplicated definition. In heve.h should be enough, so you can remove define from this file.